From: Agathe Porte <gagath@debian.org>
Subject: [PATCH] disable tcp_server feature

Due to the very sensitive nature of kanata we want to reduce the attack surface
to a minimum. Disable the tcp_server feature.

Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,6 @@
 [features]
 cmd = ["kanata-parser/cmd"]
 default = [
-    "tcp_server",
     "win_sendinput_send_scancodes",
     "zippychord",
 ]
